test/controllers/api/v2/docker_images_controller_test.rb

```diff
@@ -4,9 +4,7 @@ module Katello
   class Api::V2::DockerImagesControllerTest < ActionController::TestCase
     def models
       @repo = Repository.find(katello_repositories(:redis))
-      @image = @repo.docker_images.create!({:image_id => "abc123", :uuid => "123xyz"},
-                                           :without_protection => true
-                                          )
+      @image = @repo.docker_images.create!({:image_id => "abc123", :uuid => "123xyz"})
```

Comment:
There's no need for it anymore, its been removed in rails 4 http://stackoverflow.com/questions/21864818/activerecord-create-without-protection-wrong-number-of-arguments-2-for-1